substrate/bin/utils/chain-spec-builder/build.rs

```diff
@@ -22,4 +22,7 @@ fn main() {
 	if let Ok(profile) = env::var("PROFILE") {
 		println!("cargo:rustc-cfg=build_type=\"{}\"", profile);
 	}
+
+	#[cfg(feature = "generate-readme")]
+	println!("cargo:rerun-if-changed=README.docify.md");
```

Comment:
I played a bit with this, and surprisingly the following seems to be not required here:
```suggestion
```

Reply:
Resolved here: [6f9c200](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/6f9c200325c3480b2aab5b69507ea64d1bdf8e14)